optaplanner-website-docs/pom.xml

```diff
@@ -103,5 +104,18 @@
         <antora.playbook>antora-playbook-author.yml</antora.playbook>
       </properties>
     </profile>
+
+    <profile>
+      <id>antora-windows</id>
+      <activation>
+        <os>
+          <family>Windows</family>
+        </os>
+      </activation>
+      <properties>
+        <!-- The wrapper script node_modules/.bin/antora does not resolve the path to this file correctly. -->
+        <antora.binary>node_modules/@antora/cli/bin/antora</antora.binary>
+      </properties>
+    </profile>
```

Comment:
@Christopher-Chianelli Another thing that the antora maven plugin could do automatically to avoid boilerplate code in our pom.xml :)

Reply:
No, I think that if we fix the frontend plugin issue, all of this becomes unnecessary.

The frontend plugin should allow us to run `npx antora <options>` or `npm run build` with this in `package.json`:
```json
"scripts": {
  "build": "antora <options>"
}
```
and that should be perfectly cross-platform.

Reply:
Thanks for sharing Jiri!
@rsynek ^^ would you mind taking a look?

Reply:
I agree with @yurloc and will take a look. However, since we don't have releases of the frontend plugin under our control, it's important to keep this fix meanwhile.